src/components/bodhi-dls/cardVoting.js

```diff
@@ -200,12 +200,12 @@ class CardVoting extends Component {
     return (
       <div className="cardVoting">
         <div className="header">
-          <p style={{ marginBottom: '24px' }}><FormattedMessage id="str.RAISE" />:</p>
+          <p style={{ marginBottom: '24px' }}><FormattedMessage id="str.RAISE" defaultMessage="RAISED" />:</p>
           <h3><span>{amount.toFixed(2)}</span>
             <span>{token}</span></h3>
         </div>
         <div className="body">
-          <p style={{ marginBottom: '24px' }}><FormattedMessage id="str.outcome" /></p>
+          <p style={{ marginBottom: '24px' }}><FormattedMessage id="str.outcome" defaultMessage="OUTCOMES" /></p>
```

Comment:
why is this lowercase `str.outcome` but the one above it `str.RAISE`?

Reply:
hah, since str.raise is used for Raised, but str.RAISE is for RAISED, 
will use another pr to handle the id camelCase.

Reply:
👍